spec/support/elasticsearch.rb

```diff
@@ -0,0 +1,62 @@
+# See https://medium.com/@rowanoulton/testing-elasticsearch-in-rails-22a3296d989 .
+require 'elasticsearch/extensions/test/cluster'
+
+RSpec.configure do |config|
+
+  es_port = 9250
+  es_options = {
+    network_host: 'localhost',
+    port: es_port,
+    number_of_nodes: 1,
+    timeout: 120
+  }
+  if ENV['TEST_CLUSTER_COMMAND'].present?
+    es_options[:command] = ENV['TEST_CLUSTER_COMMAND']
+  end
+
+  searchable_models = [Notice, Entity]
+
+  # Start an in-memory Elasticsearch cluster for integration tests. Runs on
+  # port 9250 so as not to interfere with development/production clusters.
+  # This may throw a warning that the cluster is already running, but you can
+  # ignore that.
+  config.before :suite do
+    Elasticsearch::Extensions::Test::Cluster.start(**es_options) unless Elasticsearch::Extensions::Test::Cluster.running?(on: es_port)
```

Comment:
Are you using a linter or something to do the style corrections? Does that style guide allow infinitely long lines?

Reply:
I have *just* integrated a linter, but it was after I submitted this PR (and no it does not allow infinitely long lines). I'll go back and fix this now that I have my linter to guide me.